src/main/java/pingis/config/SecurityDevConfig.java

```diff
@@ -23,7 +23,7 @@ public void configure(WebSecurity web) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http
+        http.csrf().ignoringAntMatchers("**").and()
```

Comment:
I'd rather not disable CSRF protection entirely for this. IIRC the CSRF token is possible to pass using a HTTP header too, so that needs to be looked into.